siriuspy/siriuspy/devices/timing.py

```diff
@@ -191,3 +193,120 @@ def cmd_external_trigger(self):
     def is_in_inj_table(self):
         """."""
         return self.mode_str in Event.MODES[1:4]
+
+
+class Trigger(_Device):
+    """Device trigger."""
+
+    STATES = ('Dsbl', 'Enbl')
+    POLARITIES = ('Normal', 'Inverse')
+
+    def __init__(self, trigname):
+        """Init."""
+        self._database = _get_hl_trigger_database(trigname)
+        self._properties = tuple(self._database)
+        self._source_options = self._database['Src-Sel']['enums']
+        super().__init__(trigname, properties=self._properties)
+
+    @property
+    def status(self):
```

Comment:
maybe we could add a property with the description of the status? for example, if the status is ok, it would return a string "Ok", if bit 1 and 3 indicates problem, it would return a string with the description of each bit, in a comma separated form. What do you think?

Reply:
good idea, working on that

Reply:
done, please take a look!

Reply:
Very nice @anacso17 !